src/Core/OrganizationFeatures/OrganizationConnections/ValidateBillingSyncKeyCommand.cs

```diff
@@ -1,20 +1,17 @@
 ﻿using Bit.Core.Entities;
```

Comment:
This command (and interface) has been moved from the `OrganizationFeatures.OrganizationSponsorships.FamiliesForEnterprise` namespaces to the more general `OrganizationFeatures.OrganizationConnections` namespace - because it's shared between features that use the billing sync config (which is a type of organization connection).

However I wonder whether this should be an AuthenticationHandler instead, which is how we verify SCIM access: https://github.com/bitwarden/server/blob/c01d2de14389e1a95e85020fd89deb0edba87391/bitwarden_license/src/Scim/Utilities/ApiKeyAuthenticationHandler.cs.

It doesn't really fit a command, it's not an action. It's more like a helper method for authenticating the user.

Reply:
I'm not quite sure how I'd do it either to be honest. I'd rather leave it as a separate refactor than hold up this ticket for it, I just thought I'd mention the possibility.